app/src/pages/SectionHeaderLeft/SectionHeaderLeft.tsx

```diff
@@ -17,19 +18,16 @@ export const SectionHeaderLeft = ({
         <DualfiSwitcher
           type="text"
           options={options}
-          onChange={(value) => {
-            onChange(String(value));
+          onChange={(v) => {
```

Comment:
why this change? generally, v is a kind of variable name that should be avoided.